x-pack/test/ingest_manager_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
+import { warnAndSkipTest } from '../../helpers';
+
+export default function ({ getService }: FtrProviderContext) {
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const dockerServers = getService('dockerServers');
+  const log = getService('log');
+  const pkgName = 'es_assets';
+  const pkgVersion = '0.1.0';
+  const pkgKey = `${pkgName}-${pkgVersion}`;
+  const logsTemplateName = `logs-${pkgName}.test_logs`;
+  const metricsTemplateName = `metrics-${pkgName}.test_metrics`;
+
+  const deletePackage = async (pkg: string) => {
+    await supertest.delete(`/api/ingest_manager/epm/packages/${pkg}`).set('kbn-xsrf', 'xxxx');
+  };
+
+  const server = dockerServers.get('registry');
+
+  describe('installs and uninstalls all assets', async () => {
+    describe('installs all assets when installing a package for the first time', async () => {
+      it('should install the es_assets package', async function () {
+        if (server.enabled) {
+          await supertest
+            .post(`/api/ingest_manager/epm/packages/${pkgKey}`)
+            .set('kbn-xsrf', 'xxxx')
+            .expect(200);
+        } else {
+          warnAndSkipTest(this, log);
+        }
+      });
+      it('should install the ILM policy', async function () {
```

Comment:
If the actual installation is moved to a `before()` I would probable change this (and all following) to "should have installed..." instead of "should install", as there is no installation happening in the test body.